flatfs.go

```diff
@@ -54,6 +55,27 @@ const (
 	opRename
 )
 
+type initAccuracy string
+
+const (
+	exactA    initAccuracy = "initial-exact"
+	approxA   initAccuracy = "initial-approximate"
+	timedoutA initAccuracy = "initial-timed-out"
+)
+
+func combineAccuracy(a, b initAccuracy) initAccuracy {
+	if a == timedoutA || b == timedoutA {
+		return timedoutA
+	}
+	if a == approxA || b == approxA {
+		return approxA
+	}
+	if a == exactA && b == exactA {
+		return exactA
+	}
+	return ""
+}
```

Comment:
There shouldn't be any other cases.  The best I can do is if one is an empty string return the other.  If a and b disagree otherwise the is no good value to return so I will just return "".

Reply:
Then I'd return some form of "unknown".